tests/dao_tests/test_mail_kit_order_dao.py

```diff
@@ -53,38 +53,99 @@ def setUp(self):
                     "reference_number": self.mayolink_barcode,
                     "received": "2019-04-05 12:00:00",
                     "number": "12345",
-                    "patient": {"medical_record_number": "WEB1ABCD1234"},
+                    "patient": {
+                        "medical_record_number": "WEB1ABCD1234"
+                    },
                 }
             }
         }
 
         mayolinkapi_patcher = mock.patch(
             "rdr_service.dao.mail_kit_order_dao.MayoLinkApi", **{"return_value.post.return_value": self.mayolink_response}
         )
-        mayolinkapi_patcher.start()
+
+        self.mayolinkapi_patcher_start = mayolinkapi_patcher.start()
```

Comment:
The result of the `start` call is a MagicMock object, and they've been typically named after what they're mocking. What do you think of naming this something like `mock_mayolinkapi` instead of `mayolinkapi_patcher_start`?